float64 angle
float64 dangle
float64 effector_y
float64 effector_dy
float64 desired_acceleration
